pyorientAssignment.py

```diff
@@ -35,15 +35,31 @@
 # To find the minimum price, create a variable and initialize it to a very large number, 
 # then test each price to see if it is smaller than the current minimum. If it is, update 
 # the minimum variable with that price. You can do something similar to find the maximum.
-
+'''
 for record in records:
 	print record.price
+'''
+minPrice=500
+maxPrice=20000
```

Comment:
Although this might work, you should start with a very big number for the min, and a very small number for the max.